bundles/org.eclipse.urischeme/src/org/eclipse/urischeme/internal/registration/RegistrationLinux.java

```diff
@@ -57,6 +57,9 @@ public RegistrationLinux(IFileProvider fileProvider, IProcessExecutor processExe
 
 	@Override
 	public void handleSchemes(Collection<IScheme> toAdd, Collection<IScheme> toRemove) throws Exception {
+		if (supportsRegistration()) {
```

Comment:
Shouldn't this be
```
if (!supportsRegistration()) {
   return;
}
```
(in all versions, Linux, OS X, Windows)?

Reply:
sure